<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GlyphMappingChooser</class>
 <widget class="QDockWidget" name="GlyphMappingChooser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>251</width>
    <height>277</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Glyph Mapping</string>
  </property>
  <widget class="QWidget" name="content">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="relatedDataObject">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>(No object selected)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="upperContent">
       <layout class="QVBoxLayout" name="upperVerticalLayout">
        <item>
         <widget class="QListView" name="vectorsListView"/>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="propertyBrowserContainer">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="title">
        <string>(No selection)</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="propertyguizeug::PropertyBrowser" name="propertyBrowser">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>propertyguizeug::PropertyBrowser</class>
   <extends>QTreeView</extends>
   <header>propertyguizeug/PropertyBrowser.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
